Assets/__Scripts/MapEditor/Grid/Collections/NotesContainer.cs

```diff
@@ -12,16 +12,19 @@ public class NotesContainer : BeatmapObjectContainerCollection {
 
     private HashSet<Material> allNoteRenderers = new HashSet<Material>();
     private float epsilon = 0.001f;
+    public static float translucentCull = -0.001f;
```

Comment:
this hurts me and the .NET naming conventions

> DO use PascalCasing for all public member, type, and namespace names consisting of multiple words.

Reply:
![image](https://user-images.githubusercontent.com/534069/99892772-6bb64a00-2c70-11eb-917e-1ebc1f33034b.png)